src/5e-SRD-Spells.json

```diff
@@ -4447,12 +5232,16 @@
       "You cause a creature or an object you can see within range to grow larger or smaller for the duration. Choose either a creature or an object that is neither worn nor carried. If the target is unwilling, it can make a Constitution saving throw. On a success, the spell has no effect.",
       "If the target is a creature, everything it is wearing and carrying changes size with it. Any item dropped by an affected creature returns to normal size at once.",
       "Enlarge.",
-      "The target’s size doubles in all dimensions, and its weight is multiplied by eight. This growth increases its size by one category—from Medium to Large, for example. If there isn’t enough room for the target to double its size, the creature or object attains the maximum possible size in the space available. Until the spell ends, the target also has advantage on Strength checks and Strength saving throws. The target’s weapons also grow to match its new size. While these weapons are enlarged, the target’s attacks with them deal 1d4 extra damage.",
+      "The target\u00e2\u20ac\u2122s size doubles in all dimensions, and its weight is multiplied by eight. This growth increases its size by one category\u00e2\u20ac\u201dfrom Medium to Large, for example. If there isn\u00e2\u20ac\u2122t enough room for the target to double its size, the creature or object attains the maximum possible size in the space available. Until the spell ends, the target also has advantage on Strength checks and Strength saving throws. The target\u00e2\u20ac\u2122s weapons also grow to match its new size. While these weapons are enlarged, the target\u00e2\u20ac\u2122s attacks with them deal 1d4 extra damage.",
```

Comment:
This looks like some artifacts that aren't supposed to be here. They might show up in other places too.

Reply:
If you search the whole codebase for unicode characters with `\u`, you should be able to catch any places this happened.

Reply:
Oops, must be a side effect of something in my toolchain.

Reply:
Fixed!